bcgov/HOUS-permit-portal
# Add support for BC Sans font in PDFs

## Description

Add support for BC Sans font in PDFs
https://github.com/diegomura/react-pdf/issues/2675#issuecomment-1999002409

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-1074](https://hous-bssb.atlassian.net/browse/BPHH-1074)

## Steps to QA

1. As a submitter, create a permit application with a step code and submit
2. Login as a reviewer for the submitted application's jurisdiction and open the "Download application" modal. Download the zipfile and note that the PDFs use the BC Sans font (e.g. some text is bolded and for step code, the subscripts near the end of the PDF are rendered properly)

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
